pkg/apis/core/install/install_test.go

```diff
@@ -51,7 +51,7 @@ func TestResourceVersioner(t *testing.T) {
 
 func TestCodec(t *testing.T) {
 	pod := internal.Pod{}
-	// We do want to use package registered rather than testapi here, because we
+	// We do want to use package registered rather than legacyscheme here, because we
```

Comment:
Just remove this line… the comment doesn't make sense changed to legacy scheme, since we are using legacy scheme below